fenix/app/src/androidTest/java/org/mozilla/fenix/ui/SettingsDeleteBrowsingDataOnQuitTest.kt

```diff
@@ -166,8 +166,8 @@ class SettingsDeleteBrowsingDataOnQuitTest {
         }
         navigationToolbar {
         }.enterURLAndEnterToBrowser(downloadTestPage.toUri()) {
-        }.clickDownloadLink("smallZip.zip") {
-            verifyDownloadPrompt("smallZip.zip")
+        }.clickDownloadLink("executable.exe") {
+            verifyDownloadPrompt("executable.exe")
```

Comment:
I'm uneasy of us having something named `executable.exe` in the source tree regardless of it a test. Aside, I don't think we should have that as a test resource. I don't even remember what the executable was, nor is it safe to run. Was there a problem with the zip?

Reply:
Done, reverted the change.